infrastructure/kubernetes/common/monitoring/README.md

```diff
@@ -0,0 +1,82 @@
+# Kubernetes: monitoring
+
+This module contains Kubernetes deployments for monitoring the Mezo nodes.
+
+### Prerequisites
+
+- Infrastructure components for the `mezo-<environment>` GCP project created using the
+  corresponding Terraform module (make sure GKE version is >=1.29)
+- `gcloud` installed and authorized to access the `mezo-<environment>` GCP project
+- `kubectl` tool installed
+
+### Monitoring namespace
+
+All the kubernetes monitoring infrastructure lives inside the `monitoring`
+namespace. Before starting, you need to create the namespace using the
+following command:
+```Shell
+kubectl create namespace monitoring
+```
+
+### Secrets
+
+#### Grafana
+
+Secrets are used to set the default user and password to log in to
+the Grafana UI. Here are the required entries:
+- admin-password
+- admin-user
+
+Use `kubectl` to create the secrets or apply changes:
+```Shell
+kubectl create secret generic -n monitoring grafana-secret \
+  --from-literal=admin-user=<USER> \
+  --from-literal=admin-password=<PASSWORD>
+```
+
+#### Metrics Scraper
+
+One secret is used to set the configuration of the metrics scraper service. To
+create it, use the following command:
+```Shell
+kubectl create secret generic metrics-scraper-config -n monitoring \
+  --from-file=config.json=<PATH_TO_CONFIG>
+```
+
+Here's an example of the configuration:
+```
+{
+    "poll_rate": "2s",
+    "chain_id": "mezo_31611-1",
+    "nodes": [
+	{
+	    "rpc_url": "http://<MEZO_RPC_URL>:8545",
+	    "moniker": "<MEZO_NODE_MONIKER>"
+	},
```

Comment:
Nit: let's indent this properly.

Reply:
addressed in: d16d400e9ea253eb76773a875d56f6b06be3a3a1